scripts/setup.sh

```diff
@@ -100,3 +100,47 @@ else
   # Mac OS.
   export CHROME_BIN="/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
 fi
+
+# Check the version of Python. If its version does not start with 2.7, then
```

Comment:
Nit: This function is really just checking the version of Python. It might be worth noting what happens if the version is not 2.7 in the code below, where that logic exists (perhaps instead of the comment on line 127). Also, it would be good to mention that this function takes a command as input and returns true (1)/false (0) depending on whether the given command exists and that it is Python version 2.7.